docs/UserGuide.md

```diff
@@ -689,10 +689,18 @@ Delete INDEX? (yes/no)
       * If the specified `INDEX` is invalid, error message will appear.
    * If no: the specified session will not be removed from the session list.
 
-Examples:
+</div>
+
+:alert-warning: You will be exited from any currently entered session upon using the command.
+
+:green_book: Examples:
 * `deleteses 1` followed by `no` results in nothing changed.
 * `deleteses 2` followed by `yes` deletes the 2nd session in the session list.
```

Comment:
Perhaps these parts could be wrapped in their respective div boxes for better organisation of info? Noticed the same issue in other areas as well.